code/game/gamemodes/blob/theblob.dm

```diff
@@ -153,6 +153,13 @@
 	if(T.Enter(B,src))//Attempt to move into the tile
 		B.density = initial(B.density)
 		B.loc = T
+		if(double_target)
+			if(!overmind.can_buy(5))
+				overmind.last_attack = world.time
+				return
+			B.expand(double_target, 0, overmind.blob_reagent_datum.color, overmind)
+			overmind.blob_core.chemical_attack(T)
+			overmind.last_attack = world.time
```

Comment:
instead of adding all of this to `expand()` why not just have the `double_expand()` proc call `expand()` twice for the two turfs?

Reply:
Since we want to make sure it can expand in a cardinal direction before spreading cardinally again.